notebook/static/notebook/js/kernelselector.js

```diff
@@ -89,6 +99,14 @@ define([
                 )
             );
 
+            new_notebook_submenu.append(
+                $("<option>").attr("id", "kernel-selectmenu-"+ks.name)
+                    .click( function () {
+                        that.new_notebook(ks.name);
+                    })
+                    .text(ks.spec.display_name)
+            )
```

Comment:
one liner comment on what exactly this piece of code is doing!

Reply:
Removed the said piece of code since it is unnecessary appending the list of the new notebook kernel list.